openupgrade_scripts/scripts/base/14.0.1.3/pre-migrate.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 Odoo Community Association (OCA)
+# Copyright 2020 Opener B.V. <stefan@opener.am>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from openupgradelib import openupgrade
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.openupgrade_scripts.apriori import merged_modules, renamed_modules
+except ImportError:
+    renamed_modules = {}
+    merged_modules = {}
+    _logger.warn(
+        "You are using openupgrade_framework without having"
+        " openupgrade_scripts module available."
+        " The upgrade process will not work properly."
+    )
+
+
+xmlid_renames = [
+    # Module category renames were not detected by the analyze. These records
```

Comment:
For module categories, please implement instead the same script that was done in v13: https://github.com/OCA/OpenUpgrade/blob/13.0/odoo/addons/base/migrations/13.0.1.3/pre-migration.py#L752, to avoid some issues

Reply:
Would this work? https://github.com/OCA/openupgradelib/pull/219

Reply:
Obviously, something like also that could work.

Reply:
OK, we'll continue there then.

Reply:
Added `allow_merge` parameter and singled out module categories explicitely (although no other ids need renaming).